### PR TITLE
Trainer client tab

### PR DESCRIPTION
**Pull request by jaba9531**
- Client tab displays feed of cards with client names
- Live search filter trims down feed to respective matches
- Trainer can add an unregistered user via modal popup

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" type="text/css" href="./style/main.css"/>
   <link rel="shortcut icon" href="">
+  <link rel="stylesheet" type="text/css" href="./style/clientTab.css">
```
Comment (DChan0319) [inline client/dist/index.html:9]: Look at how we are importing our "new styles" in main.css follow that convention. Remove this line in index.html and import it the same way in main.css

File: client/src/components/dashPage/clientTabModal.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  handleUnregClientFormChange(e) {
+    this.setState({unregClientForm: e.target.value});
+  }
+
+  render() {
+    return (
+      <div>{
+        <div id="myModal" className="modal">
```
Comment (DChan0319) [inline client/src/components/dashPage/clientTabModal.jsx:30]: Just a naming thing. Not a big deal but why use myModal? what if we want to distinguish the name of the modal should name it specific to what type of modal it is for example...AddNewClientModal. MyModal is very vague and hard to figure out what the modal is actually for.

Comment (zgallagher08) [reply client/src/components/dashPage/clientTabModal.jsx:30]: agreed

File: client/src/containers/dashPageContainer.jsx [hunk 3]
```diff
@@ -53,7 +53,8 @@ class DashPageContainer extends Component {
 const mapStateToProps = function(state) {
   return {
     authenticated: state.auth.authenticated,
-    userInfo: state.auth.user
+    userInfo: state.auth.user,
+    clientList: state
```
Comment (DChan0319) [inline client/src/containers/dashPageContainer.jsx:57]: how is the clientList the whole redux store?? are you creating a whole new reducer to grab the current "trainers" clients? That reducer should be state.NAME_OF_REDUCER.clients or something like that. Not sure how you got the whole store to be a list of clients.

**Review by DChan0319 — COMMENTED** [review]
Confused about your mapStateToProps({ ClientList: state }); 
modal id naming convention not a big deal but should be changed for clarification of what that modal is actually for. 
Styling sheet should not be linked directly in the index.html but instead imported in the main.css.

**Comment by jaba9531** [comment]
clientList: state actually shouldn't be there, I'll remove it. See
mapStateToProps in containers/clientTabContainer.

File: client/dist/index.html (hunk 1 above)
Comment (jaba9531) [reply client/dist/index.html:9]: Fixed, thanks

File: client/src/components/dashPage/clientTabModal.jsx (hunk 2 above)
Comment (jaba9531) [reply client/src/components/dashPage/clientTabModal.jsx:30]: Renamed to addNewClientModal

**Pull request closed without merging**